article.Rmd

```diff
@@ -48,7 +50,7 @@ Most pop-up active transport infrastructure can be classified into three broad c
 1. 'filtered permeability', e.g. as [shown](https://twitter.com/CatrionaSwanson/status/1258322956595453952) in [@salfordcitycouncil_salford_2020]
 
 2. banning cars and to pedestrianise streets, as in New York's 'Open Streets' scheme [@litman_pandemicresilient_2020]
-3. the reallocation of one or more lanes on wide roads to create pop-up cycleways [@orsman_covid_2020]. 
+3. the reallocation of one or more lanes on wide roads to create pop-up cycleways and pavements [@orsman_covid_2020].
```

Comment:
Important point. I also plan to add a 4th one to make the distinction between reallocating space on 'spare lanes' and reallocating space on  roads with 2 lanes to make roads one-way for cars. Thoughts?

Reply:
Personally I would keep them as one. I think it's starting to end up a rather too specialised list, the more that these categories get broken down.